packages/web/src/components/navbar.tsx

```diff
@@ -0,0 +1,308 @@
+      <g clipPath="url(#clip0_1790_16713)">
+        <path
+          fill="#8E979D"
+          d="M9.522 6.776L15.475 0h-1.41l-5.17 5.882L4.767 0H.007l6.241 8.895L.007 16h1.41l5.457-6.213L11.232 16h4.76M1.926 1.04h2.166l9.972 13.969h-2.167"
+        ></path>
+      </g>
+      <defs>
+        <clipPath id="clip0_1790_16713">
+          <path fill="#fff" d="M0 0H15.987V16H0z" transform="translate(.007)"></path>
+        </clipPath>
+      </defs>
+    </svg>
+  );
+}
+
+export function Navbar() {
+  return (
+    <header className="h-12 w-full flex items-center justify-between fixed bg-background z-50 border-b border-border text-sm">
+      <nav className="flex items-center justify-between px-4 flex-1">
+        <div className="flex items-center gap-6">
+          <NavLink to="/">
+            <h1 className="font-mono font-bold flex items-center space-x-[10px]">
+              <SrcbookLogo size={20} />
+              <span>Srcbooks</span>
+            </h1>
+          </NavLink>
+
+          <ul className="flex items-center gap-6">
+            <li>
+              <NavLink
+                to="/secrets"
+                className="font-semibold text-tertiary-foreground visited:text-tertiary-foreground hover:text-foreground transition-colors"
+              >
+                Secrets
+              </NavLink>
+            </li>
+            <li>
+              <NavLink
+                to="/settings"
+                className="font-semibold text-tertiary-foreground visited:text-tertiary-foreground hover:text-foreground transition-colors"
+              >
+                Settings
+              </NavLink>
+            </li>
+            <li>
+              <a
+                href="https://hub.srcbook.com"
+                target="_blank"
+                className="font-semibold text-tertiary-foreground visited:text-tertiary-foreground hover:text-foreground transition-colors"
+              >
+                <div className="flex items-center gap-1.5">
+                  Hub
+                  <ExternalLinkIcon size={14} />
+                </div>
+              </a>
+            </li>
+          </ul>
+        </div>
+
+        <LightDarkModeDebugChanger />
+
+        <div className="flex items-center gap-2">
+          <a href="https://x.com/srcbookcom" target="_blank">
+            <Button variant="icon" size="icon" className="active:translate-y-0">
+              <SocialXIcon />
+            </Button>
+          </a>
+          <a href="https://discord.gg/shDEGBSe2d" target="_blank">
+            <Button variant="icon" size="icon" className="active:translate-y-0">
+              <SocialDiscordIcon />
+            </Button>
+          </a>
+          <a href="https://github.com/srcbookdev/srcbook" target="_blank">
+            <Button variant="icon" size="icon" className="active:translate-y-0">
+              <SocialGithubIcon />
+            </Button>
+          </a>
+        </div>
```

Comment:
I think I saw in the mockups that the `SocialXIcon` entry should no longer be here? If I got that right I'll remove it.

Reply:
Removed.